e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+    let otherUser;
+
+    beforeEach(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                otherUser = user2;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2368 Fixed Width setting should not scroll pop and display posts properly', () => {
+        // # Creating some post to verify scroll pop and Posts views
+        cy.postMessage('This is the first post');
+        Cypress._.times(18, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
+        cy.postMessage('This is the last post');
```

Comment:
It's not specifically stated in the test case but like I mentioned in our DM, "scroll pop" is an internal term that refers to a sudden change of height from initial to final render that causes each post to change render position over time, and is noticeable when some posts have image, link preview, markdown in-line image and others.  With that, please add posting of message with:
- file attachment -- each post for `jpg-image-file.jpg`, `gif-image-file.gif`, `mp3-audio-file.mp3`. Note that these file names are already in the fixture folder.
- link preview using `https://www.bbc.com/news/uk-wales-45142614`, and
- markdown image using `![gif](http://i.giphy.com/xNrM4cGJ8u3ao.gif)`.

For these to open previews, you'll need to update config with `EnableLinkPreviews: true` (see other spec files for reference).
Also, please use taller view port, maybe 1300x2000, so many posts are visible within the window.

Reply:
:heavy_check_mark: 
I have added image, link preview, markdown in-line image in posts
Verifying before and after height of these posts